README.md

```diff
@@ -5,104 +5,27 @@
-     * docker
-       * A tool to help you build and run apps within containers
-       
-   These are only required if you are also building the Codewind Index (export CODEWIND_INDEX=true)
-     * python3
-       * Python is a general-purpose interpreted, interactive, object-oriented, and high-level programming language
-     * pyyaml
-       * YAML parser and emitter for python (pip3 install pyyaml)
-
-   There are several environment variables that need to be set up. These are required in order to correctly build the collections.
-   ```bash
-   # Organization for images
-   export IMAGE_REGISTRY_ORG=kabanero
-
-   # Whether to build the Codewind Index file
-   export CODEWIND_INDEX=false
-   ```
-
-1. Build collections.
-
-   From the base directory, run the build script.  For example:
-    ```
-    . ./ci/build.sh
-    ```
-
-   Note that this will build all the collections in the incubator directory.
-
-   Following the build, you can find the generated collection assets in the file://$PWD/ci/assets/ directory and all the docker images in your local docker registry.
-
-## Testing the collections locally
-
-1. Add the local collections repository to Appsody.
-
-   The `./ci/build.sh` should have added the local kabanero index into the Appsody repository list. You can check this by running the command:
-   ```
-   appsody repo list
-   ```
-   
-   The output from this command should include entries similar to:
-   ``` 
-   appsodyhub              https://github.com/appsody/stacks/releases/latest/download/incubator-index.yaml        
-   kabanero-index          file:///Users/myuser/kabanero-io/collections/ci/assets/kabanero-index.yaml      
-   kabanero-index-local    file:///Users/myuser/kabanero-io/collections/ci/assets/kabanero-index-local.yaml
-   ```
-   
-   If the `kabanero-index-local` repository is not in the list then it can be added to Appsody using the Appsody CLI:
-
-   ```bash
-   appsody repo add kabanero-index-local file://$PWD/ci/assets/kabanero-index-local.yaml
-   ```
-   This will enable you to do an `appsody init` for a collection that is in the newly built kabanero collections.
-
-1. Test the collections.
-
-   To test the collections using the local docker images, rather than pulling them from docker hub, set the following environment variable:
-   ```
-   export APPSODY_PULL_POLICY=IFNOTPRESENT
-   ```
-   
-   The `appsody init` will now use the local docker images. For example:
-   ```
-   appsody init kabanero-index-local/java-microprofile
-   ```
-   
-   will create a new java-microprofile project using the local docker images and the local collection artifacts. 
-
-1. Test the pipelines and other components that have been included in the collection within the Kabanero/OpenShift environment.
-
-## [Optional] Test the collections using a test GIT release
+In addition to the Working with Kabanero Collections guide the following section provides information about *"Testing the collections using a test GIT release"*
```

Comment:
It might be nice to have a header after this to delimit the section?

Reply:
Resolved. Added a header.